### DISCUSSION TITLE
Carousel AutoPay is not added

### DISCUSSION THREAD

**Discussion by Jannatul13064** [saadeghi/daisyui#2716] [Ideas / feature requests]
Is there any easy way to use the carousel autoplay option?

**Comment by dominikkrulak** [comment]
You'd have to build a JS function/package that manages an autoplay option.

Daisyui does not use any javascript to manage/manipulate a state of a component.
This thing is up to you.

Anyway, I've recently started building carousel's autoplay|play|pause and scroll to next|previous slide but I haven't finished it yet. If I don't forget I can post the package repository here.

**Reply by HappyCodingCoding** [reply]
Look forward to see you releasing this plugin.

**Reply by damsfx** [reply]
@dominikkrulak any news?

**Reply by dominikkrulak** [reply]
I've got it working in my local setup already, but not ready yet for publication. When? I don't know exactly. Sometimes in summer, this year.
